app/views/movies/index.html.erb

```diff
@@ -12,7 +12,9 @@
         </p>
         <p class="card-text">
           <% movie.screenings.each do |screening| %>
-            <%= link_to "#{screening.start.strftime('%d %b, %H:%M')}", new_screening_reservation_path(screening), class: 'btn btn-warning btn-sm' %>
+            <%= link_to "#{screening.start.strftime('%d %b, %H:%M')}",
```

Comment:
```suggestion
            <%= link_to "#{format_date(screening.start)",
```

If you have it in multiple places you can create function inside app/helpers/application_helper with for example for example:
``
format_date(date)
  date.strftime('%d %b, %H:%M')
end
``
You can also create it for showing only hours or something and this way time will be more consistent through the app and if any changes it will be easier to change 🥇